styles/styles.css

```diff
@@ -1173,8 +1197,8 @@ main.blue-contract .section.section-with-title p {
 
 .redesign-v2 .icon svg {
   fill: currentcolor;
-  height: 1em;
-  width: 1em;
+  height: 1rem;
```

Comment:
The `1em` for height and width was intentional, so the icon will have the same size as the text next to it. With `1rem` the icon will be always the same if it is used in the heading (ex: h1) or in a regular paragraph.
rem - text size: 50px but icon is still 1rem

<img width="651" alt="obraz" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/125962117/57530c72-15fa-4d1d-8915-70db86acfb23">

vs 
em font size ratio between text and icon is the same all the time : 

<img width="655" alt="obraz" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/125962117/6763ce95-66b9-479a-ac79-c0440ae2f39c">

Reply:
updated